tests/validation/config.json

```diff
@@ -1,1220 +1,1276 @@
+            "excludedEngines": ["webgl1"],
+            "referenceImage": "prepass-ssao-highlight-layer.png"
+        },
+        {
+            "title": "Prepass SSAO + point light",
+            "renderCount": 10,
+            "playgroundId": "#XDNVAY#6",
+            "excludedEngines": ["webgl1"],
+            "referenceImage": "prepass-ssao-point-light.png"
+        },
+        {
+            "title": "Prepass SSAO + on/off post-process",
+            "renderCount": 10,
+            "playgroundId": "#1VI6WV#20",
+            "excludedEngines": ["webgl1"],
+            "referenceImage": "prepass-ssao-on-off-pp.png"
+        },
+        {
+            "title": "Prepass SSAO + thin instances",
+            "renderCount": 10,
+            "playgroundId": "#V1JE4Z#25",
+            "excludedEngines": ["webgl1"],
+            "referenceImage": "prepass-ssao-thin-instances.png"
+        },
+        {
+            "title": "Prepass SSAO + depth renderer",
+            "renderCount": 10,
+            "playgroundId": "#3HPMAA#1",
+            "excludedEngines": ["webgl1"],
+            "referenceImage": "prepass-ssao-depth-renderer.png"
+        },
+        {
+            "title": "Prepass SSAO + visibility",
+            "renderCount": 30,
+            "playgroundId": "#PXC9CF#4",
+            "excludedEngines": ["webgl1"],
+            "referenceImage": "prepass-ssao-visibility.png"
+        },
+        {
+            "title": "Prepass SSAO + default pipeline",
+            "renderCount": 10,
+            "playgroundId": "#NAW8EA#7",
+            "excludedEngines": ["webgl1"],
+            "referenceImage": "prepass-ssao-default-pipeline.png"
+        },
+        {
+            "title": "Prepass MBlur + Lens",
+            "renderCount": 10,
+            "playgroundId": "#ZEB7H6#23",
+            "excludedEngines": ["webgl1"],
+            "referenceImage": "prepass-mb-lens.png"
+        },
+        {
+            "title": "Prepass SSAO + MSAA",
+            "renderCount": 10,
+            "playgroundId": "#12MKMN#7",
+            "excludedEngines": ["webgl1"],
+            "referenceImage": "prepass-ssao-msaa.png"
+        },
+        {
+            "title": "Scissor test",
+            "renderCount": 10,
+            "playgroundId": "#W7E7CF#12",
+            "excludedEngines": ["webgl1"],
+            "referenceImage": "scissor-test.png"
+        },
+        {
+            "title": "Refraction local cube map STD",
+            "renderCount": 1,
+            "playgroundId": "#RJN64I#7",
+            "referenceImage": "refraction-local-std.png"
+        },
+        {
+            "title": "Refraction local cube map PBR",
+            "renderCount": 1,
+            "playgroundId": "#RJN64I#5",
+            "referenceImage": "refraction-local-pbr.png"
+        },
+        {
+            "title": "Instances + motion blur",
+            "renderCount": 10,
+            "playgroundId": "#YB006J#402",
+            "excludedEngines": ["webgl1"],

```

Comment:
You should also exclude the **webgpu** engine of all the "instances + motion blur" tests as we are currently limited to only 8 vertex buffers.